board_mod.py

```diff
@@ -20,7 +20,7 @@ def add_message(self, message):
 
 	def print_header(self, fo):
 		fo.write("/*******************************************************************/\n")
-		fo.write("/*                                                        {}                                                                        */\n".format(self.name))
+		fo.write("/*      {}      */\n".format(self.name))
```

Comment:
Nit: On peut utiliser de l'interpolation de string

```suggestion
		fo.write(f"/*      {self.name}      */\n")
```